src/main/java/world/bentobox/border/Settings.java

```diff
@@ -11,146 +11,82 @@
-    @ConfigComment("To disable addon it is necessary to write its name in new line that starts with -. Example:")
-    @ConfigComment("disabled-gamemodes:")
-    @ConfigComment(" - BSkyBlock")
     @ConfigEntry(path = "disabled-gamemodes")
     private Set<String> disabledGameModes = new HashSet<>();
 
-    @ConfigComment("")
-    @ConfigComment("Use vanilla world border. Requires WorldBorderAPI plugin.")
-    @ConfigComment("Download from https://github.com/yannicklamprecht/WorldBorderAPI/releases")
     @ConfigEntry(path = "use-wbapi")
-    private boolean useWbapi = false;
+    private boolean useWbapi = true;
 
-    @ConfigComment("")
-    @ConfigComment("Teleport players back inside the border if they somehow get outside.")
-    @ConfigComment("This will teleport players back inside if they toggle the border with a command.")
     @ConfigEntry(path = "return-teleport")
     private boolean returnTeleport = true;
 
-    @ConfigComment("")
-    @ConfigComment("Only applies if WBAPI isn't used.")
-    @ConfigComment("Use barrier blocks. If false, the border is indicated by particles only.")
     @ConfigEntry(path = "use-barrier-blocks")
     private boolean useBarrierBlocks = true;
 
-    @ConfigComment("")
-    @ConfigComment("Default border behavior")
     @ConfigEntry(path = "show-by-default")
-    private boolean showByDefault= true;
+    private boolean showByDefault = true;
 
-    @ConfigComment("")
-    @ConfigComment("Only applies if WBAPI isn't used.")
-    @ConfigComment("Show max-protection range border. This is a visual border only and not a barrier.")
     @ConfigEntry(path = "show-max-border")
-    private boolean showMaxBorder= true;
+    private boolean showMaxBorder = true;
 
-    @ConfigComment("")
-    @ConfigComment("Only applies if WBAPI isn't used.")
-    @ConfigComment("Enables/disables all types of wall particles shown by the addon")
     @ConfigEntry(path = "show-particles")
     private boolean showParticles = true;
 
-    /**
-     * @param disabledGameModes new disabledGameModes value.
-     */
-    public void setDisabledGameModes(Set<String> disabledGameModes)
-    {
+    public void setDisabledGameModes(Set<String> disabledGameModes) {
         this.disabledGameModes = disabledGameModes;
     }
 
-    /**
-     * @return disabledGameModes value.
-     */
-    public Set<String> getDisabledGameModes()
-    {
+    public Set<String> getDisabledGameModes() {
         return this.disabledGameModes;
     }
 
-    /**
-     * @return the useBarrierBlocks
-     */
+    public boolean isUseWbapi() {
+        return useWbapi;
+    }
+
+    public void setUseWbapi(boolean useWbapi) {
+        this.useWbapi = useWbapi;
+    }
+
+    public boolean isReturnTeleport() {
+        return returnTeleport;
+    }
+
+    public void setReturnTeleport(boolean returnTeleport) {
+        this.returnTeleport = returnTeleport;
+    }
+
     public boolean isUseBarrierBlocks() {
         return useBarrierBlocks;
     }
 
-    /**
```

Comment:
There's not reason to delete the javadocs. They can provide explanation in the IDE when required